src/services/school/model.js

```diff
@@ -26,6 +27,15 @@ const SCHOOL_OF_DELETED_USERS = { name: 'graveyard school (tombstone users only)
 
 const defaultFeatures = [];
 
+const defaultStudentList =
```

Comment:
please move this into a hook of the schools service.
Business logic should be kept out of the datalayer as much as possible. In the old server, it should be placed either directly in a service where possible, or in a hook of a service otherwise.